source/publishing_data/02_harvesting_a_catalog/harvesters/ftp_with_meta_csv.rst

```diff
@@ -79,6 +79,11 @@ Accepted metadata columns
    * * Semantic (if activated)
      * ``semantic.rml_mapping``, ``semantic.classes``, ``semantic.properties``
 
+- The ``geographic_reference_auto`` column contains the Boolean value ``true`` if the **Geographic coverage** metadata for the dataset is set to **Automatic**.
+- The ``geographic_reference`` column contains the location if the **Geographic coverage** metadata is set to **Specific**.
```

Comment:
If this column is supposed to set the name of the location, we must indicate how/where to find the value of the locations name

Reply:
Also, we may provide additional information about the format expected here.
The value for `geographic_reference` must be a list of string, where string are georeference uids.  It's often a list of only one element, e.g. the coverage of "Arbres parisiens" is Paris (https://public.opendatasoft.com/api/v2/catalog/datasets/arbresremarquablesparis2011) but it can also be multivaluated, that's why a list is expected in all cases.

Examples of values:
  * `["fr_80_75056", "fr_80_60159"]` for a dataset coverage over "Paris" and "Compiègne" cities
  * `["fr_80_75056"]` for a dataset coverage over "Paris" city